### PR TITLE
release: v2.13.3

### PR DESCRIPTION
**Pull request by bk2204**
This is an in-progress look at the next release of Git LFS, v2.13.3, which is scheduled for Friday, March 26, 2021.

We're publishing these changes early so that folks on @git-lfs/implementers can check that things work with their various platforms.

I've attached some builds below for people to use for testing:

[git-lfs-darwin-amd64-v2.13.3-pre0.zip](https://github.com/git-lfs/git-lfs/files/6185168/git-lfs-darwin-amd64-v2.13.3-pre0.zip)
[git-lfs-darwin-arm64-v2.13.3-pre0.zip](https://github.com/git-lfs/git-lfs/files/6185169/git-lfs-darwin-arm64-v2.13.3-pre0.zip)
[git-lfs-freebsd-386-v2.13.3-pre0.tar.gz](https://github.com/git-lfs/git-lfs/files/6185172/git-lfs-freebsd-386-v2.13.3-pre0.tar.gz)
[git-lfs-freebsd-amd64-v2.13.3-pre0.tar.gz](https://github.com/git-lfs/git-lfs/files/6185173/git-lfs-freebsd-amd64-v2.13.3-pre0.tar.gz)
[git-lfs-linux-386-v2.13.3-pre0.tar.gz](https://github.com/git-lfs/git-lfs/files/6185174/git-lfs-linux-386-v2.13.3-pre0.tar.gz)
[git-lfs-linux-amd64-v2.13.3-pre0.tar.gz](https://github.com/git-lfs/git-lfs/files/6185175/git-lfs-linux-amd64-v2.13.3-pre0.tar.gz)
[git-lfs-linux-arm64-v2.13.3-pre0.tar.gz](https://github.com/git-lfs/git-lfs/files/6185176/git-lfs-linux-arm64-v2.13.3-pre0.tar.gz)
[git-lfs-linux-arm-v2.13.3-pre0.tar.gz](https://github.com/git-lfs/git-lfs/files/6185177/git-lfs-linux-arm-v2.13.3-pre0.tar.gz)
[git-lfs-linux-ppc64le-v2.13.3-pre0.tar.gz](https://github.com/git-lfs/git-lfs/files/6185178/git-lfs-linux-ppc64le-v2.13.3-pre0.tar.gz)
[git-lfs-linux-s390x-v2.13.3-pre0.tar.gz](https://github.com/git-lfs/git-lfs/files/6185179/git-lfs-linux-s390x-v2.13.3-pre0.tar.gz)
[git-lfs-v2.13.3-pre0.tar.gz](https://github.com/git-lfs/git-lfs/files/6185180/git-lfs-v2.13.3-pre0.tar.gz)
[git-lfs-windows-386-v2.13.3-pre0.zip](https://github.com/git-lfs/git-lfs/files/6185181/git-lfs-windows-386-v2.13.3-pre0.zip)
[git-lfs-windows-amd64-v2.13.3-pre0.zip](https://github.com/git-lfs/git-lfs/files/6185182/git-lfs-windows-amd64-v2.13.3-pre0.zip)

In addition, the CI system will produce artifacts for Windows, Linux, and macOS if you prefer to use those; they should be equivalent.

/cc @git-lfs/core
/cc @git-lfs/implementers
/cc @git-lfs/releases

### CODE REVIEW

**Comment by chrisd8088** [comment]
If #4449 looks reasonable, and if there's a chance it could be included in this release, I think that would be valuable for those with long Git histories and who are using both stashes and the `git lfs prune` command.  But I realize it might not be possible to include it for one reason or another (e.g., if there's a better approach the problem).

**Comment by bk2204** [comment]
> If #4449 looks reasonable, and if there's a chance it could be included in this release, I think that would be valuable for those with long Git histories and who are using both stashes and the `git lfs prune` command. But I realize it might not be possible to include it for one reason or another (e.g., if there's a better approach the problem).

I think that should be fine. I've approved that PR and I can rebuild this commit once it's merged.  I originally gave myself more than the standard two days because Wednesday and Thursday are potentially busy for me, but that turns out to have been very convenient since we can now include your change.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
 # Git LFS Changelog
 
-## 2.13.2 (13 Jan 2020)
+## 2.13.3 (26 Mar 2021)
+
+This release fixes a bug that caused `git lfs prune` to hang, updates some
```
Comment (chrisd8088) [inline CHANGELOG.md:5]: If we add #4449 to this release we should tweak this to note that it fixes two bugs of this type (sigh).

**Review by chrisd8088 — APPROVED** [review]
LGTM, thank you for pulling this together!

**Comment by dohertyfjatl** [comment]
We've tested this version against Bitbucket Server's test matrix and it passes ✅

**Comment by dbast** [comment]
Test git-lfs-darwin-arm64-v2.13.3-pre0.zip on Apple M1 (darwin-arm64) hardware. Binary works directly works without any signature issue or warning etc.  ✅

**Comment by bk2204** [comment]
I've cherry-picked in #4449 here as well and updated the changelog, and I'll include some new binaries below if folks want to test.

[git-lfs-darwin-amd64-v2.13.3-pre1.zip](https://github.com/git-lfs/git-lfs/files/6207595/git-lfs-darwin-amd64-v2.13.3-pre1.zip)
[git-lfs-darwin-arm64-v2.13.3-pre1.zip](https://github.com/git-lfs/git-lfs/files/6207596/git-lfs-darwin-arm64-v2.13.3-pre1.zip)
[git-lfs-freebsd-386-v2.13.3-pre1.tar.gz](https://github.com/git-lfs/git-lfs/files/6207597/git-lfs-freebsd-386-v2.13.3-pre1.tar.gz)
[git-lfs-freebsd-amd64-v2.13.3-pre1.tar.gz](https://github.com/git-lfs/git-lfs/files/6207598/git-lfs-freebsd-amd64-v2.13.3-pre1.tar.gz)
[git-lfs-linux-386-v2.13.3-pre1.tar.gz](https://github.com/git-lfs/git-lfs/files/6207599/git-lfs-linux-386-v2.13.3-pre1.tar.gz)
[git-lfs-linux-amd64-v2.13.3-pre1.tar.gz](https://github.com/git-lfs/git-lfs/files/6207600/git-lfs-linux-amd64-v2.13.3-pre1.tar.gz)
[git-lfs-linux-arm64-v2.13.3-pre1.tar.gz](https://github.com/git-lfs/git-lfs/files/6207602/git-lfs-linux-arm64-v2.13.3-pre1.tar.gz)
[git-lfs-linux-arm-v2.13.3-pre1.tar.gz](https://github.com/git-lfs/git-lfs/files/6207603/git-lfs-linux-arm-v2.13.3-pre1.tar.gz)
[git-lfs-linux-ppc64le-v2.13.3-pre1.tar.gz](https://github.com/git-lfs/git-lfs/files/6207605/git-lfs-linux-ppc64le-v2.13.3-pre1.tar.gz)
[git-lfs-linux-s390x-v2.13.3-pre1.tar.gz](https://github.com/git-lfs/git-lfs/files/6207607/git-lfs-linux-s390x-v2.13.3-pre1.tar.gz)
[git-lfs-v2.13.3-pre1.tar.gz](https://github.com/git-lfs/git-lfs/files/6207608/git-lfs-v2.13.3-pre1.tar.gz)
[git-lfs-windows-386-v2.13.3-pre1.zip](https://github.com/git-lfs/git-lfs/files/6207609/git-lfs-windows-386-v2.13.3-pre1.zip)
[git-lfs-windows-amd64-v2.13.3-pre1.zip](https://github.com/git-lfs/git-lfs/files/6207610/git-lfs-windows-amd64-v2.13.3-pre1.zip)

The release is still on for tomorrow, but if someone needs more time to test and would like me to push it off, please shout.

**Pull request merged**